pulley/src/interp.rs

```diff
@@ -1694,6 +1694,50 @@ impl OpVisitor for Interpreter<'_> {
         ControlFlow::Continue(())
     }
 
+    #[inline]
+    fn push_frame_save(&mut self, amt: u32, regs: RegSet<XReg>) -> ControlFlow<Done> {
+        // Decrement the stack pointer `amt` bytes plus 2 pointers more for
+        // fp/lr.
+        let ptr_size = size_of::<usize>();
+        let full_amt = usize::try_from(amt).unwrap() + 2 * ptr_size;
```

Comment:
Shouldn’t we be saving the full 64 bits of each register, not the usize, for when we are on 32-bit platforms? do/lr seem fine to be usize but not GPRs.

Reply:
Oh the `ptr_size` bits are just for fp/lr which are pointer-sized, the `amt` which Cranelift decides handles the 8-bytes-per-integer-register